src/cargo/util/style.rs

```diff
@@ -11,3 +15,14 @@ pub const NOTE: Style = AnsiColor::Cyan.on_default().effects(Effects::BOLD);
 pub const GOOD: Style = AnsiColor::Green.on_default().effects(Effects::BOLD);
 pub const VALID: Style = AnsiColor::Cyan.on_default().effects(Effects::BOLD);
 pub const INVALID: Style = AnsiColor::Yellow.on_default().effects(Effects::BOLD);
+
+
+/// Write a note to the given writer.
```

Comment:
I am not sure if the note function should be put into here. Does this make sense? Do you have a better place for it?